src/Epic/canRender.ts

```diff
@@ -6,10 +6,11 @@ import { BrazeMessageProps } from './index';
 export const canRenderEpic = (brazeMessageProps: BrazeMessageProps): boolean => {
     const { buttonText, buttonUrl, ophanComponentId } = brazeMessageProps;
     const paragraphs = parseParagraphs(brazeMessageProps);
+    console.log(buttonText, buttonUrl, ophanComponentId, paragraphs);
     return Boolean(buttonText && buttonUrl && ophanComponentId && paragraphs.length > 0);
 };
 
-const parseParagraphs = (brazeMessageProps: BrazeMessageProps): string[] => {
+export const parseParagraphs = (brazeMessageProps: BrazeMessageProps): string[] => {
```

Comment:
Now that we're not tied to the interface of the RRCP `ContributionsEpic`, I wonder if it makes sense to have the interface of our component more closely match the way the data will be in Braze (i.e. `paragraph1`, `paragraph2` etc). What do you reckon?

Reply:
I agree, however this function is still useful as it allows us to go from paragraph1, paragraph2 to a list of paragraphs, which makes the render and canRender functions simpler.